tests/testthat/test-ticktext-linebreaks.R

```diff
@@ -0,0 +1,64 @@
+context("axes")
+
+# Compute margin
+comp_margin <- function(gg, axisTickText) {
+  plot <- ggfun("plot_clone")(gg)
+  theme <- ggfun("plot_theme")(plot)
+  elements <- names(which(sapply(theme, inherits, "element")))
+  for (i in elements) {
+    theme[[i]] <- ggplot2::calc_element(i, theme)
+  }
+  
+  pm <- unitConvert(theme$plot.margin, "pixels")
+  gglayout <- list(
+    margin = list(t = pm[[1]], r = pm[[2]], b = pm[[3]], l = pm[[4]])
+  )
+  axisTitle <- theme[["axis.title.y"]]
+  axisObj <- list(
+    ticktext = "djdfkjdfdklj",
+    tickfont = text2font(theme[["axis.text"]], "width"),
+    ticklen = unitConvert(theme$axis.ticks.length, "pixels", "width")
+  )
+  
+  gglayout$margin[["l"]] + axisObj$ticklen +
+    bbox(axisTickText, 0, axisObj$tickfont$size)[["width"]] +
+    bbox("y", 90, unitConvert(axisTitle, "pixels", "width"))[["width"]]
+}
+
+expect_margin <- function(L, gg, ticktext) {
+  margin_l <- comp_margin(gg, ticktext)
+  expect_equal(round(L$layout$margin$l, 10), round(margin_l, 10))
+}
```

Comment:
I appreciate you going down this road, but just FYI, I think the visual tests would have been sufficient here.

Reply:
As I just started with the contributing and testing business this was a good exercise. (;